inc/live-editor.php

```diff
@@ -76,21 +76,21 @@ function post_metadata( $value, $post_id, $meta_key ) {
 	function frontend_scripts() {
 		wp_enqueue_script(
 			'live-editor-front',
-			plugin_dir_url( __FILE__ ) . '../js/live-editor/live-editor-front' . SITEORIGIN_PANELS_JS_SUFFIX . '.js',
+			siteorigin_panels_url( 'js/live-editor/live-editor-front' ) . SITEORIGIN_PANELS_JS_SUFFIX . '.js',
```

Comment:
Here especially, I think we'd want to have everything passed into `siteorigin_panels_url` if possible.